element/src/commonMain/kotlin/RootElement.kt

```diff
@@ -1,16 +1,67 @@
 package com.juul.krayon.element
 
+import com.juul.krayon.kanvas.IsPointInPath
 import com.juul.krayon.kanvas.Kanvas
+import com.juul.krayon.kanvas.Transform
+import com.juul.krayon.kanvas.Transform.Translate
 
 public class RootElement : Element() {
 
     override val tag: String get() = "root"
 
+    public var onClickFallback: (() -> Unit)? by attributes.withDefault { null }
```

Comment:
It isn't quite clear to me why the "fallback"? What does it do/offer beyond the `onClick` property lambda?

Reply:
I'll add some documentation. The goal was that this is the "fallback" behavior when a touch occurs that doesn't hit any other clickable element. As an example of using it, if sections of a chart have click events to select them, you might use the fallback to deselect.

I almost called it `onBackgroundClick` but that didn't feel accurate. This event is still fired if you click on rendered elements that don't have `onClick` listeners, but that's not really the "background."

Any ideas on a name? Open to suggestions.

Reply:
The name is fine as-is ("fallback"), just a simple code comment with a similar explanation to what you gave here would be great (I believe it would be sufficient to eliminating confusion).